scripts/update-integration-test-coordinates.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+python integration-test.py -printcoords | python scripts/update-integration-test-coordinates.py
```

Comment:
I think so. That way it doesn't come as a surprise if someone tries to run it out of the `scripts/` dir, or we end up in a `$PWD` that we didn't expect.